sdk/storage/azure-storage-blob/tests/test_common_blob.py

```diff
@@ -247,6 +247,34 @@ def test_create_blob_with_lease_id(self, resource_group, location, storage_accou
         content = blob.download_blob(lease=lease).readall()
         self.assertEqual(content, data)
 
+    @GlobalStorageAccountPreparer()
+    def test_create_blob_with_generator(self, resource_group, location, storage_account, storage_account_key):
```

Comment:
I'm wondering if the content size that the generator could produce is larger than chunk size, it is still working.
let's say if the chunk size is 4096, the generator generate "hello" every time, and the generator could generate 820 "hello" in total. Not sure if it is working. I'm also checking

Reply:
Will be fixed in #8513